### ISSUE TITLE
Officially maintain Flatpak on Flathub

### ISSUE THREAD

**Issue by bertob** [teeworlds/teeworlds#1932]
[There is a version of Teeworlds on Flathub](https://flathub.org/apps/details/com.teeworlds.Teeworlds), but it's not up to date and doesn't seem to be maintained by upstream. Officially maintaining it upstream would be a very easy way to make the newest version of Teeworlds available to users on all distributions.

Flathub repository: https://github.com/flathub/com.teeworlds.Teeworlds/
Developer guide: https://github.com/flathub/flathub/wiki/App-Submission

**Comment by barthalion** [comment]
@Dune-jr, seeing you assigned now, would you be interested in getting write access to Flathub repository? It has been added back when we weren't as strict about contacting upstream first as now.

This means we ship our own [appdata file](https://github.com/flathub/com.teeworlds.Teeworlds/blob/master/com.teeworlds.Teeworlds.appdata.xml), adding OARS content ratings and information about releases, and desktop file, although apparently for no good reason looking at the one in your repository). I also patched bam project file to support aarch64 but I didn't test it on actual hardware.

**Comment by Dune-jr** [comment]
Yes, @barthalion; in fact, I wrote a mail yesterday, but I don't think it went through the filters (yet).

> I also patched bam project file to support aarch64 but I didn't test it on actual hardware.

Yeah, I saw that. I was on my way to propose to apply the patch, at least to our local fork https://github.com/teeworlds/bam

**Comment by barthalion** [comment]
Ah, indeed. Being part of Teeworlds organization sounds like a good proof. Unfortunately GitHub doesn't allow to add entire org to contributors, but I can simply add current members.

**Comment by Dune-jr** [comment]
No need to add the whole org. @oy is our BDFL, so add him as well. Probably best to add our admin @minus7 as well just in case.

**Comment by barthalion** [comment]
Done, make sure to accept the invitations and let me know if you have any questions.

**Comment by Dune-jr** [comment]
Since @minus7 declined, could you add @heinrich5991 instead? Much thanks :)

**Comment by barthalion** [comment]
Absolutely, invitation sent.
